compiler/src/dotty/tools/dotc/transform/ExpandPrivate.scala

```diff
@@ -38,6 +41,9 @@ class ExpandPrivate extends MiniPhaseTransform with IdentityDenotTransformer { t
 
   override def phaseName: String = "expandPrivate"
 
+  // This phase moves methods around(in infortransform) so it may need to make other methods public
+  override def runsAfter: Set[Class[_ <: Phase]] = Set(classOf[MoveStatics])
```

Comment:
As pointed out by @smarter , there is a typo here: `infortransform`. Otherwise, LGTM.